Flag page redesign

bitWolfy opened the pull request:
This is a redesign of the flag page that I mocked up and proposed a few months ago.

![flagpage update](https://user-images.githubusercontent.com/1503448/135674638-f65fb06d-a6f4-4b7b-9b17-fddbe1e50725.png)

This adds detailed explanations of what each flag reason means specifically, as well as streamline the whole page somewhat.
My hope is that this will reduce the number of people who use flags incorrectly.

Since the header text is imported from a [wiki page](https://e621.net/wiki_pages/help:flag_notice), rather than specified in the settings, that page will have to be edited manually to complete the change if this pull request is accepted.
```
[quote]
h3. Abusing this system by willfully flagging posts for incorrect reasons may lead to disciplinary actions against your account.
[/quote]

Select an option below that most closely matches the reason why you believe this post should be deleted. Keep in mind that personal dislike of specific themes that may appear in images – such as gore, rape, or cub – are not valid reasons to flag the post on their own. Use the [[e621:blacklist|blacklist]] to filter out images that contain these tags.

If a reason is not listed here, check the options listed on the Report page.

If you are the owner of this submission, and would like to have it deleted (and not just replaced with an updated version), then please file a "takedown request":/static/takedown instead.
```

Commit:
[Flags] Redesign the new flag page

Commit:
[Flags] Tweaks to the 48 hour deletion reason

kiranoot reviewed (approved):
:100: lgtm @Earlopain this look good to you too?

Earlopain commented on config/danbooru_default_config.rb:
```diff
@@ -649,14 +649,47 @@ def enable_signups?
       ]
     end
+    def flag_reason_48hours
+      "If you are the artist, and want this image to be taken down [b]permanently[/b], file a \"takedown\":/static/takedowns instead.\nTo replace the image with a \"fixed\" version, upload that image first, and then use the \"Duplicate or inferior version\" reason above.\nFor accidentally released paysite or private content, use the \"Paysite, commercial, or private content\" reason above."
```
```suggestion
      "If you are the artist, and want this image to be taken down [b]permanently[/b], file a \"takedown\":/static/takedown instead.\nTo replace the image with a \"fixed\" version, upload that image first, and then use the \"Duplicate or inferior version\" reason above.\nFor accidentally released paysite or private content, use the \"Paysite, commercial, or private content\" reason above."
```
The current link 404s, the same is true for the wiki page template. Once that's fixed this should be all good.

bitWolfy replied:
Should be fixed in 0c99e1209507bf633783b3e2b8214ef0ec50b855.

Commit:
[Flags] Fix a configuration error

Rebasing the branch went wrong

Commit:
[Flags] Re-arrange the config values for clarity

48 hour flag reason ended up under the deletion reasons because of a rebase gone awry

Commit:
[Flags] Fix a typo on the 48 hour reason explanation
